src/admin/resolvers/queries/Collection.integration.ts

```diff
@@ -210,11 +214,40 @@ describe('admin queries: Collection', () => {
         author,
         status: CollectionStatus.PUBLISHED,
       });
-      await createCollectionHelper(db, {
+      const fakeCollection = await createCollectionHelper(db, {
         title: 'the dude abides man',
         author,
         status: CollectionStatus.PUBLISHED,
       });
+
+      // create label and collection-label association
+      fakeLabel = await createLabelHelper(db, 'region-east-africa', 'fakeUser');
+      const collectionLabelInputData: CreateCollectionLabelInput = {
+        collectionId: fakeCollection.id,
+        labelId: fakeLabel.id,
+        createdAt: new Date(),
+        createdBy: 'fakeUser',
+      };
+      await createCollectionLabelHelper(db, collectionLabelInputData);
+    });
+
+    it('should search by labelExternalIds', async () => {
+      const { data } = await server.executeOperation({
+        query: SEARCH_COLLECTIONS,
+        variables: {
+          filters: {
+            labelExternalIds: [fakeLabel.externalId],
+          },
+        },
+      });
+
+      const collections = data?.searchCollections?.collections;
+
+      expect(collections.length).to.equal(1);
+      expect(collections[0].title).to.equal('the dude abides man');
+      expect(collections[0].labels[0].externalId).to.equal(
+        fakeLabel.externalId
+      );
```

Comment:
Can we add another test case for when a collection has more than one label and we search based on only one?